test/libethereum/unit.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file unit.cpp
+ * @author Dimitry Khokhlov <Dimitry@ethdev.com>
+ * @date 2015
+ * libethereum unit test functions.
+ */
+
+#include <boost/filesystem/operations.hpp>
+#include <boost/test/unit_test.hpp>
+#include <libethereum/Defaults.h>
+#include <libethereum/AccountDiff.h>
+
+BOOST_AUTO_TEST_SUITE(libethereum)
+
+BOOST_AUTO_TEST_CASE(AccountDiff)
+{
+	std::cout << "AccountDiff" << std::endl;
```

Comment:
No logs please. Test name can be requested by passing a param to Boost.Test from command line.